tests/integration_tests/test_empty_value_handling.py

```
@@ -153,93 +165,97 @@ def _test_upsert_empty_metadata(self, collection):
-        
+
+        collection.upsert(ids=test_ids, documents=updated_docs, metadatas=updated_metas)
+
         # Verify all values were stored correctly
         results = collection.get(ids=test_ids, include=["documents", "metadatas"])
         assert len(results["ids"]) == 4, "Should find all 4 results"
-        
-        for i, (expected_doc, expected_meta) in enumerate(zip(updated_docs, updated_metas)):
+
+        for i, (expected_doc, expected_meta) in enumerate(
+            zip(updated_docs, updated_metas)
```

Ruff (0.14.10)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)